File: src/main/java/yaasl/server/model/Location.java
package yaasl.server.model;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

import static javax.persistence.GenerationType.AUTO;

@Entity
public class Location {

    private Long id;
    private String icao;
    private String name;

    public Location() {
    }

    @Id
    @GeneratedValue(strategy=AUTO)
    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getIcao() {
        return icao;
    }

    public void setIcao(String icao) {
        this.icao = icao;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

}

File: src/main/kotlin/yaasl/server/model/Location.kt
package yaasl.server.model

import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType.AUTO
import javax.persistence.Id

@Entity
class Location(@get:Id
               @get:GeneratedValue(strategy = AUTO)
               var id: Long? = null,
               var icao: String? = null,
               var name: String? = null)
